scripts/deploy.sh

```diff
@@ -197,7 +197,7 @@ else
     error $? "Could not pushd /data/socorro"
     PYTHONPATH=. ../socorro-virtualenv/bin/python \
         ../socorro-virtualenv/bin/alembic \
-        -c config/alembic.ini upgrade head &> /var/log/socorro/alembic.log
+        -c config/alembic.ini upgrade heads &> /var/log/socorro/alembic.log
```

Comment:
In the past, the details have been shown in the log if there is a failure.. now i am unsure why tho? :D